extensions/libraries/objectrepository/objectrepository.lcb

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright (C) 2017 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+/**
+This library provides support persistent object references within a session
+*/
+
+library com.livecode.library.objectrepository
+
+use com.livecode.engine
+
+metadata version is "1.0.0"
+metadata author is "LiveCode"
+metadata title is "Object Repository Library"
+metadata userVisible is "false"
+
+variable mSequence as Integer
+
+variable mObjects as Array
+
+/**
+
+Store an object in the object repository and get its ID
```

Comment:
I think the handler description should be within the "Description:" sections. I'm not sure these comments will appear in the documentation

Reply:
I'm pretty sure this is treated as the `Summary` element